tests/testsuite/package.rs

```diff
@@ -5213,22 +5295,23 @@ fn workspace_with_local_deps_packaging_one_fails() {
         .build();
 
     // We can't package just level1, because there's a dependency on level2.
-    p.cargo("package -p level1")
+    p.cargo("package -p level1 -Zpackage-workspace")
+        .masquerade_as_nightly_cargo(&["package-workspace"])
         .replace_crates_io(crates_io.index_url())
         .with_status(101)
-        .with_stdout("")
-        .with_stderr(
+        .with_stdout_data("")
```

Comment:
Note that if the test is very intentionally wanting blank output on a stream, we don't put `str![]` on it